docusaurus/docs/troubleshooting.mdx

```diff
@@ -0,0 +1,10 @@
+---
+pagination_prev: null
+pagination_next: null
+---
+
+# Troubleshooting
+
+## When using two different CI providers, why are not all test files executed?
+
+If you are utilizing two CI providers simultaneously, for example, CircleCI and TravisCI, and you run a test suite, it is necessary to have a separate API token for tests executed on CircleCI and another separate API token for the test suite executed on TravisCI.
```

Comment:
Maybe we should add the links for the troubleshootings of each library at the bottom?

Similar to what we have in the welcome page:

<img width="855" alt="Screenshot 2024-02-26 at 16 09 59" src="https://github.com/KnapsackPro/docs.knapsackpro.com/assets/5238698/823c2b10-fe75-4400-938b-4afb0a830006">

Reply:
Does this look ok?
<img width="1029" alt="Troubleshooting___Knapsack_Pro_Docs" src="https://github.com/KnapsackPro/docs.knapsackpro.com/assets/3391962/64ce48c6-335f-47a4-b73c-67a9196a0357">

Reply:
I actually based it on the index page and this part: `Can't see your language/testing framework above?` But libraries will work even better.